packages/gestalt/package.json

```diff
@@ -42,7 +42,6 @@
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-replace": "^2.2.0",
     "rollup-plugin-visualizer": "^3.3.2",
-    "rollup-watch": "^4.3.1",
```

Comment:
Rollup added its own `watch` method in `0.46` - so we haven't used this package in a while.